saleor/site/models.py

```diff
@@ -16,6 +16,9 @@
 from .error_codes import SiteErrorCode
 from .patch_sites import patch_contrib_sites
 
+if TYPE_CHECKING:
+    from ..settings import DEFAULT_FROM_EMAIL
+
```

Comment:
Does it make sense to solve weird fail of mypy for `sender_address = settings.DEFAULT_FROM_EMAIL  # type: ignore` line?

Reply:
You placed it under the TYPE_CHECKING, I don't think that this will be accesible in line 120.

Reply:
Yep, and it also means that there is a lack of a test for the scenario without `sender_address`. Could you add one?

Reply:
I added the test, however can't see any usage of the function in the codebase.